mysql-test/suite/innodb/t/innodb_onlinealter_encryption-master.opt

```diff
@@ -0,0 +1,4 @@
+$KEYRING_PLUGIN_OPT
```

Comment:
Tests that require keyring plugins shoud be extracted into include files. There should be a test in innodb testsuite that loads keyring_file and sources the test as include file and there should be a test in keyring_vault  that loads keyring_vault plugin and sources the include file. For exmple see table_encypt_2 test case.